scripts/serve-local.sh

```diff
@@ -2,8 +2,8 @@
 
 set -o errexit
 
-CURRENT_DIR=`dirname $BASH_SOURCE`
-source $CURRENT_DIR/docker-helpers.sh
+D=`dirname $BASH_SOURCE`
```

Comment:
I tend to prefer more explicit names, any significant reason to shorten this one?

Reply:
is long. I started volunteering on another django project and I switched my naming convention. But CURRENT_DIR isn't hurting anyone. It stays.